app/views/test/components/mentoring/mentor_inboxes/show.html.haml

```diff
@@ -6,4 +6,4 @@
   = select_tag :tracks_endpoint_state, options_for_select(%w[Success Error Loading], params[:tracks_endpoint_state])
   = submit_tag "Submit", name: nil
 %hr/
-= mentor_inbox({ endpoint: test_components_mentoring_mentor_inbox_conversations_path, query: { state: params[:conversations_endpoint_state], sort: :exercise }, retry: { retry: params[:conversations_endpoint_state] == "Loading" } }, { endpoint: test_components_mentoring_mentor_inbox_tracks_path, query: { state: params[:tracks_endpoint_state] }, retry: { retry: params[:tracks_endpoint_state] == "Loading" } }, @sort_options)
+= mentor_inbox({ endpoint: test_components_mentoring_mentor_inbox_conversations_path, query: { state: params[:conversations_endpoint_state], sort: :exercise }, options: { retry: params[:conversations_endpoint_state] == "Loading" } }, { endpoint: test_components_mentoring_mentor_inbox_tracks_path, query: { state: params[:tracks_endpoint_state] }, options: { retry: params[:tracks_endpoint_state] == "Loading" } }, @sort_options)
```

Comment:
When https://github.com/exercism/v3-website/pull/82 is merged, this needs to be updated.